src/components/LetterInputs.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { update } from "../reducers/word";
+import { increment } from "../reducers/Increment";
+import { useSelector } from "react-redux";
+
+function LetterInputs({ wordLength }) {
+  const [letterStates, setLetterStates] = useState(Array(wordLength).fill(""));
+
+  const updateLetterState = (index, value) => {
+    const updatedStates = [...letterStates];
+    updatedStates[index] = value;
+    setLetterStates(updatedStates);
+  };
+
+  const clearInputFields = () => {
+    const clearedStates = Array(wordLength).fill("");
+    setLetterStates(clearedStates);
+  };
+
+  const dispatch = useDispatch();
+  const counterValue = useSelector((state) => state.increment.value);
+
+  return (
+    <>
+      <div style={{ display: "flex", alignItems: "center", gap: "1rem" }}>
+        {Array.from({ length: wordLength }, (_, i) => (
```

Comment:
I've definitely never ween this notation, would you mind explaining it with a comment?

Reply:
Specifically `(_, i)`